### ISSUE TITLE
App crashes during verification

### ISSUE THREAD

**Issue by vorletzter** [vector-im/element-android#2774]
**Describe the bug**
A clean build of v.1.0.15 crashes after signing in during the "is this you?"-Process in Android < 11

**To Reproduce**
1) Start with a fresh install
2) Sign in
3) The app crashes

**Smartphone (please complete the following information):**
 - Device: Emulator (API 28 and API 23)
 - Works with Emulator API 30 and Physical Android 11 Device.

**Additional context**
Might be because of changes in
[23623b8895958acfe9b8d6e8b5b674e7d012fe33](https://github.com/vector-im/element-android/commit/23623b8895958acfe9b8d6e8b5b674e7d012fe33)
or an older version of [WindowInsetsControllerCompat](https://developer.android.com/reference/androidx/core/view/WindowInsetsControllerCompat)

But i have no real Experience with Android Development. So please bear that in mind :)

Debug 

```
E/VectorUncaughtExceptionHandler: FATAL EXCEPTION Element Build : 40100153
    Element Version : 1.0.15 [40100153] (F-ce6d4c4a-2021-02-03 17:08:27 +0100)
    SDK Version : 0.0.1 (ce6d4c4a)
    Phone : AOSP on IA Emulator (5875966 9 REL)
    Memory statuses 
    usedSize   13 MB
    freeSize   1 MB
    totalSize   15 MB
    Thread: main, Exception: io.reactivex.exceptions.UndeliverableException: The exception could not be delivered to the consumer because it has already canceled/disposed the flow or the exception has nowhere to go to begin with. Further reading: https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling | java.lang.NoSuchMethodError: No virtual method getWindowInsetsController()Landroid/view/WindowInsetsController; in class Landroid/view/View; or its super classes (declaration of 'android.view.View' appears in /system/framework/framework.jar!classes2.dex)
        at io.reactivex.plugins.RxJavaPlugins.onError(RxJavaPlugins.java:367)
        at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:126)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.NoSuchMethodError: No virtual method getWindowInsetsController()Landroid/view/WindowInsetsController; in class Landroid/view/View; or its super classes (declaration of 'android.view.View' appears in /system/framework/framework.jar!classes2.dex)
        at im.vector.app.features.popup.PopupAlertManager.clearLightStatusBar(PopupAlertManager.kt:169)
        at im.vector.app.features.popup.PopupAlertManager.showAlert(PopupAlertManager.kt:186)
        at im.vector.app.features.popup.PopupAlertManager.showAlert$default(PopupAlertManager.kt:185)
        at im.vector.app.features.popup.PopupAlertManager.displayNextIfPossible(PopupAlertManager.kt:156)
        at im.vector.app.features.popup.PopupAlertManager.access$displayNextIfPossible(PopupAlertManager.kt:45)
        at im.vector.app.features.popup.PopupAlertManager$postVectorAlert$2.run(PopupAlertManager.kt:57)
        at android.app.Activity.runOnUiThread(Activity.java:6282)
        at im.vector.app.features.popup.PopupAlertManager.postVectorAlert(PopupAlertManager.kt:56)
        at im.vector.app.features.home.HomeActivity.promptSecurityEvent(HomeActivity.kt:302)
        at im.vector.app.features.home.HomeActivity.handleOnNewSession(HomeActivity.kt:252)
        at im.vector.app.features.home.HomeActivity.access$handleOnNewSession(HomeActivity.kt:72)
        at im.vector.app.features.home.HomeActivity$onCreate$2.invoke(HomeActivity.kt:150)
        at im.vector.app.features.home.HomeActivity$onCreate$2.invoke(HomeActivity.kt:72)
        at im.vector.app.core.platform.VectorBaseActivity$observeViewEvents$1.accept(VectorBaseActivity.kt:114)
        at im.vector.app.core.platform.VectorBaseActivity$observeViewEvents$1.accept(VectorBaseActivity.kt:92)
        at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:63)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:201)
        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:255)
        at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:124)
        at android.os.Handler.handleCallback(Handler.java:873) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6669) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
```

**Comment by bahur142** [comment]
I also noted this kind of crash when sign-in with a new user. Happened on Android 6.0.1.
Also it crashes when sign-out. It is impossible to sign-out from this version.

But the crash happen only when theme is White Theme (or when default theme is system theme). If you change to Black or Dark, the problem is gone.

**Comment by bmarty** [comment]
Fixed in 1.0.16, thanks for the heads up!
